mmseg/apis/train.py

```diff
@@ -105,6 +105,8 @@ def train_segmentor(model,
             dist=distributed,
             shuffle=False)
         eval_cfg = cfg.get('evaluation', {})
+        eval_cfg['pre_eval'] = cfg.runner[
+            'pre_eval'] if 'pre_eval' in cfg.runner else False
```

Comment:
Why we need to get `pre_eval` form `cfg.runner`?

Reply:
MMDet3d use train api with eval hook when training. However, the dataset of mmdet3d doesn't implement `pre_eval` function. So, we set pre_eval False in default when using training api and use schedule to set pre_eval True.